src/test/node/node-runtime-skipfiles-skip-node-internals.txt

```diff
@@ -0,0 +1,13 @@
+<anonymous> @ ${workspaceFolder}/simpleNode/index.js:1:1
```

Comment:
Should this file be deleted now that the custom assert is used?

Reply:
Yep! Removed